### PR TITLE
feat: add Inworld AI TTS provider (WebSocket)

### PR DESCRIPTION
**Pull request by coval-cale**
Adds an Inworld AI TTS provider (our first), covering the two GA Realtime TTS 1.5 models: 1.5 Max and 1.5 Mini.

## Summary by CodeRabbit
* **New Features**
  * Added an Inworld text-to-speech provider with support for `inworld-tts-1.5-max` and `inworld-tts-1.5-mini`.
  * Updated the UI to recognize the new provider/models, including consistent color mapping and normalized display names.
* **Bug Fixes**
  * Improved audio assembly by stripping per-chunk WAV wrappers to produce clean PCM output.
* **Tests**
  * Added streaming and edge-case coverage, including auth URL validation, key redaction on errors, and missing/unsupported model handling.
* **Chores**
  * Added Inworld API key configuration and updated model registry entries (kept inactive for current orchestration).

<h3>Greptile Summary</h3>

This PR introduces Inworld AI as a new TTS provider, wiring up a WebSocket bidirectional streaming client for the two GA Realtime TTS 1.5 models (`inworld-tts-1.5-max` and `inworld-tts-1.5-mini`). The implementation handles per-chunk WAV unwrapping, API key scrubbing from error messages, and is backed by a thorough test suite.

- **New provider (`inworld.py`)**: Connects via WebSocket, sends a `create` + `send_text` message pair, assembles base64-decoded PCM from streamed JSON chunks, strips per-chunk RIFF headers, and delegates to `finalize_tts_result` for TTFA calculation and WAV output.
- **Registry entries**: Both models are added to `MODEL_REGISTRY` with `status=_RETIRED`, which conflicts with the frontend display code added in the same PR and the "GA" framing in the PR description — see the inline comment.
- **Frontend**: Indigo-family colors and normalized display names are registered for both models and the `"Inworld AI"` provider.

<h3>Confidence Score: 4/5</h3>

Safe to merge with one fix: both Inworld models are registered as RETIRED, meaning they will never be benchmarked and will be hidden from the site despite the frontend display code added in this same PR.

The provider implementation, tests, config, and frontend display code are all solid. The one concrete defect is in models.py: both new models use _RETIRED, which according to the registry docstring means 'not benchmarked, hidden even if old data exists.' This directly contradicts the PR goal of adding a live GA provider and makes the frontend colors/formatters additions dead code. Everything else — WebSocket streaming logic, WAV header stripping, key redaction, test coverage — is well-executed.

runner/src/coval_bench/registries/models.py — the status field on both Inworld entries needs to be reconsidered before the models will actually appear or run.

<h3>Important Files Changed</h3>

| Filename | Overview |
|----------|----------|
| runner/src/coval_bench/providers/tts/inworld.py | New Inworld WebSocket TTS provider — logic is correct and well-tested; minor fragility in WAV subchunk detection. |
| runner/src/coval_bench/registries/models.py | Both Inworld models registered as RETIRED (hidden, never benchmarked) while frontend display code was added — likely a copy-paste mistake from the adjacent Soniox entry. |
| runner/tests/providers/tts/test_inworld.py | Comprehensive test suite covering happy path, WAV header stripping, auth URL encoding, key redaction in errors, all models, empty audio, streamed error messages, and missing API key. |
| runner/src/coval_bench/config.py | Adds inworld_api_key: SecretStr | None = None to Settings — straightforward and consistent with all other provider keys. |
| web/lib/config/colors.ts | Adds indigo-family model and provider colors for Inworld AI — consistent with the existing color-coding pattern. |
| web/lib/utils/formatters.ts | Adds normalizeModelName and normalizeTTSProviderName entries for Inworld models — correct and consistent. |

</details>

<h3>Sequence Diagram</h3>

<a href="#gh-light-mode-only">

```mermaid
%%{init: {'theme': 'neutral'}}%%
sequenceDiagram
    participant Bench as Benchmark Runner
    participant Provider as InworldTTSProvider
    participant WS as Inworld WebSocket API

    Bench->>Provider: synthesize(text)
    Provider->>WS: "connect(wss://...?authorization=Basic%20<key>)"
    Provider->>WS: "send(create: {voiceId, modelId, audioConfig})"
    Provider->>WS: "send(send_text: {text, flush_context})"
    loop Stream audio chunks
        WS-->>Provider: "JSON {result: {audioChunk: {audioContent: <b64>}}}"
        Provider->>Provider: base64.b64decode → _pcm_from_chunk (strip RIFF header)
    end
    WS-->>Provider: "JSON {result: {flushCompleted: {}}}"
    Provider->>Provider: finalize_tts_result (TTFA + WAV write)
    Provider-->>Bench: TTSResult(audio_path, ttfa_ms, ...)
```

</a>
<a href="#gh-dark-mode-only">

```mermaid
%%{init: {'theme': 'base', 'themeVariables': {"darkMode": true, "background": "#0d1117", "primaryColor": "#21262d", "primaryTextColor": "#e6edf3", "primaryBorderColor": "#8b949e", "lineColor": "#8b949e", "textColor": "#e6edf3", "edgeLabelBackground": "#161b22", "actorBkg": "#21262d", "actorBorder": "#8b949e", "actorTextColor": "#e6edf3", "actorLineColor": "#8b949e", "signalColor": "#8b949e", "signalTextColor": "#e6edf3", "noteBkgColor": "#373320", "noteBorderColor": "#d4a72c", "noteTextColor": "#f0e6c0", "labelBoxBkgColor": "#21262d", "labelBoxBorderColor": "#8b949e", "labelTextColor": "#e6edf3", "loopTextColor": "#e6edf3", "activationBkgColor": "#30363d", "activationBorderColor": "#8b949e"}}}%%
sequenceDiagram
    participant Bench as Benchmark Runner
    participant Provider as InworldTTSProvider
    participant WS as Inworld WebSocket API

    Bench->>Provider: synthesize(text)
    Provider->>WS: "connect(wss://...?authorization=Basic%20<key>)"
    Provider->>WS: "send(create: {voiceId, modelId, audioConfig})"
    Provider->>WS: "send(send_text: {text, flush_context})"
    loop Stream audio chunks
        WS-->>Provider: "JSON {result: {audioChunk: {audioContent: <b64>}}}"
        Provider->>Provider: base64.b64decode → _pcm_from_chunk (strip RIFF header)
    end
    WS-->>Provider: "JSON {result: {flushCompleted: {}}}"
    Provider->>Provider: finalize_tts_result (TTFA + WAV write)
    Provider-->>Bench: TTSResult(audio_path, ttfa_ms, ...)
```

</a>

<sub>Reviews (4): Last reviewed commit: ["Strip Inworld per-chunk WAV headers from..."](https://github.com/coval-ai/benchmarks/commit/86197dd7198835f17fa73412987f639dae51cc3d) | [Re-trigger Greptile](https://app.greptile.com/api/retrigger?id=37747487)</sub>

### CODE REVIEW

File: runner/src/coval_bench/providers/tts/inworld.py [hunk 1]
```diff
@@ -0,0 +1,147 @@
+# Copyright 2026 The Coval Benchmarks Authors
+# SPDX-License-Identifier: Apache-2.0
+
+"""Inworld AI TTS provider — WebSocket bidirectional streaming.
+
+Wire protocol:
+  connect wss://api.inworld.ai/tts/v1/voice:streamBidirectional
```
Comment (seribaymadina) [inline runner/src/coval_bench/providers/tts/inworld.py]: non-blocking: up to you but we might wanna clean up the obvious info in this docstring

File: runner/src/coval_bench/providers/tts/inworld.py [hunk 2]
```diff
@@ -0,0 +1,147 @@
...
+        )
+
+        try:
+            async with ws_client.connect(url) as ws:
+                start = time.monotonic()
+                await ws.send(create_msg)
+                await ws.send(text_msg)
+
+                async for raw in ws:
+                    if isinstance(raw, bytes):
```
[the remark this answers is not in the document]
Comment (coval-cale) [reply runner/src/coval_bench/providers/tts/inworld.py:103]: Will add in the coming provider refactor.

File: runner/src/coval_bench/providers/tts/inworld.py [hunk 3]
```diff
@@ -0,0 +1,150 @@
...
+        first_chunk_at: float | None = None
+
+        auth_param = quote(f"Basic {self._api_key}", safe="")
+        url = f"{_WS_URL}?authorization={auth_param}"
+        create_msg = json.dumps(
+            {
+                "create": {
+                    "voiceId": self._voice,
+                    "modelId": self._model,
+                    "audioConfig": {"audioEncoding": "LINEAR16", "sampleRateHertz": SAMPLE_RATE},
```
Comment (seribaymadina) [inline runner/src/coval_bench/providers/tts/inworld.py:84]: they might carry their own wav headers that will break ttfa and wer;

Comment (seribaymadina) [reply runner/src/coval_bench/providers/tts/inworld.py:84]: for other providers we request raw "PCM"

**Review by seribaymadina — CHANGES_REQUESTED** [review]

File: runner/src/coval_bench/providers/tts/inworld.py (hunk 3 above)
Comment (coval-cale) [reply runner/src/coval_bench/providers/tts/inworld.py:84]: Your reviews rock! Epic thanks @seribaymadina !

Comment (coval-cale) [reply runner/src/coval_bench/providers/tts/inworld.py:84]: Fixed!

**Review by seribaymadina — APPROVED** [review]
excited to see the results in the benchmarks!!
